src/main/java/com/amazonaws/services/neptune/io/StreamThrottle.java

```diff
@@ -28,7 +28,7 @@ public class StreamThrottle {
     private volatile int tumblingWindowSize = 10;
 
     private static final long MAX_QUEUE_HIGH_WATERMARK = 10000;
-    private static final long QUEUE_SIZE_BYTES = 10000000;
+    private static final long QUEUE_SIZE_BYTES = 1000000000;
```

Comment:
Is there any potential negative impact here as well? Is there any possibility of overwhelming a small stream?

Reply:
No possibility of overwhelming stream, because the producer buffers records: these values control when backpressure from the producer cause the export to pause puts to KInesis. Increasing the values encourages a steadier throughput, with fewer pauses.